addons/portal/controllers/portal.py

```diff
@@ -167,6 +186,100 @@ def _prepare_portal_overview_values(self):
             }]
         }
 
+    def _is_own_partner_address(self, partner, address_id):
```

Comment:
Instead of "my own address", isn't it more something like ``_can_update_address`` ? Implementation details (own or children invoice / delivery) are hidden but purpose is globally, may I update it.